etc/conda3_packages-linux-64.yml

```diff
@@ -25,10 +25,10 @@ dependencies:
   - binutils_linux-64=2.34=hc952b39_18
   - blosc=1.18.1=he1b5a44_0
   - boost=1.70.0=py37h9de70de_1
-  - boost-cpp=1.70.0=h8e57a91_2
+  - boost-cpp=1.70.0=ha2d47e9_1
```

Comment:
Double checking this right now, not sure why we dropped a build.

Reply:
Related to a downgrade of icu for the older matplotlib, icu is only an dependency (host,run) for matplotib on linux.